src/fides/api/models/privacy_center_config.py

```diff
@@ -0,0 +1,106 @@
+            "description_subtext": [
+                "In order to opt-out of certain consent preferences, we may need to identify your account via your email address. This is optional."
+            ],
+            "icon_path": "/consent.svg",
+            "identity_inputs": {"email": "optional"},
+            "title": "Manage your consent",
+            "modalTitle": "Manage your consent",
+            "confirmButtonText": "Continue",
+            "cancelButtonText": "Cancel",
+        },
+        "page": {
+            "consentOptions": [
+                {
+                    "fidesDataUseKey": "marketing.advertising",
+                    "name": "Data Sales or Sharing",
+                    "description": "We may use some of your personal information for behavioral advertising purposes, which may be interpreted as 'Data Sales' or 'Data Sharing' under regulations such as CCPA, CPRA, VCDPA, etc.",
+                    "url": "https://example.com/privacy#data-sales",
+                    "default": {"value": True, "globalPrivacyControl": False},
+                    "highlight": False,
+                    "cookieKeys": ["data_sales"],
+                    "executable": False,
+                },
+                {
+                    "fidesDataUseKey": "marketing.advertising.first_party",
+                    "name": "Email Marketing",
+                    "description": "We may use some of your personal information to contact you about our products & services.",
+                    "url": "https://example.com/privacy#email-marketing",
+                    "default": {"value": True, "globalPrivacyControl": False},
+                    "highlight": False,
+                    "cookieKeys": ["tracking"],
+                    "executable": False,
+                },
+                {
+                    "fidesDataUseKey": "functional",
+                    "name": "Product Analytics",
+                    "description": "We may use some of your personal information to collect analytics about how you use our products & services.",
+                    "url": "https://example.com/privacy#analytics",
+                    "default": True,
+                    "highlight": False,
+                    "cookieKeys": ["analytics"],
+                    "executable": False,
+                },
+            ],
+            "description": "Manage your consent preferences, including the option to select 'Do Not Sell or Share My Personal Information'.",
+            "description_subtext": [
+                "When you use our services, you're trusting us with your information. We understand this is a big responsibility and work hard to protect your information and put you in control."
+            ],
+            "policy_key": "default_consent_policy",
+            "title": "Manage your consent",
+        },
+    },
+}
+
+
+class PrivacyCenterConfig(Base):
```

Comment:
I generally like to see at least one test on at the model level - if you end up moving your `create_or_update` logic here that would be a good thing to test!

Reply:
I added a test now that I have the `create_or_update` function on the model.